config/packages/prod/monolog.yaml

```diff
@@ -2,24 +2,27 @@ monolog:
     handlers:
         main:
             type: fingers_crossed
-            action_level: error
-            handler: nested
-            excluded_404s:
-                # regex: exclude all 404 errors from the logs
-                - ^/
-        nested:
-            type: stream
-            path: "%kernel.logs_dir%/%kernel.environment%.log"
-            level: debug
+            action_level: critical
+            handler: main_handler
+        main_handler:
+            type: group
+            members: [console, log_file, mailer]
         console:
             type: console
+            level: debug
             process_psr_3_messages: false
-            channels: ["!event", "!doctrine"]
-        deprecation:
+        log_file:
             type: stream
-            path: "%kernel.logs_dir%/%kernel.environment%.deprecations.log"
-        deprecation_filter:
-            type: filter
-            handler: deprecation
-            max_level: info
-            channels: ["php"]
+            level: info
+            path: '%kernel.logs_dir%/foo_%kernel.environment%.log'
+        mailer:
+            type: deduplication
+            handler: swift
+        swift:
+            type: swift_mailer
+            level: critical
+            from_email: '%env(MAILER_FROM_EMAIL)%'
+            to_email: '%env(MAILER_TO_EMAIL)%'
+            subject: '[tools.svgtranslate] Error: %%message%%'
```

Comment:
@samwilson `tools.svgtranslate` :/ I'm wondering if there's a `%kernel.root_dir%` or something like that, which we can probably assume is the name of the application